packages/open-api-gateway/package.json

```diff
@@ -23,7 +23,7 @@
     "pre-compile": "npx projen pre-compile",
     "release:mainline": "npx projen release:mainline",
     "test": "npx projen test",
-    "test:integration": "npx projen test:integration",
+    "test-integration": "npx projen test-integration",
```

Comment:
nitpick: why hyphenate?

Reply:
I found I couldn't run it standalone from the root of my project with the colon, ie:

`npx nx run @aws-prototyping-sdk/open-api-gateway:test:integration` failed with something like "no such target integration"

Reply:
Ahh you need to wrap with quotes i.e. `npx nx run @aws-prototyping-sdk/open-api-gateway:"test:integration"`

Reply:
Gotcha! Will switch it back :)